lambda/models/lambda_functions.py

```diff
@@ -36,16 +36,16 @@
 app.add_middleware(AWSAPIGatewayMiddleware)
 
 
-@app.post(path="", status_code=status.HTTP_200_OK, include_in_schema=False)
-@app.post(path="/", status_code=status.HTTP_200_OK)
+@app.post(path="", status_code=status.HTTP_201_CREATED, include_in_schema=False)
```

Comment:
Let's still keep this as a 200- since this is going to be an asynchronous process to stand everything up, we just need to return 200 that we received and processed the request enough to start getting the infrastructure set up in the background. Technically we don't know if a resource correctly created or not yet.